src/index.html

```diff
@@ -17,6 +17,93 @@
     />
   </head>
   <body>
+    <header class="menu">
+      <div class="item logo">
+        <a href="#">
+          <img
+            src="images/logo.png"
+            alt="Logo"
+          />
+        </a>
+      </div>
+
+      <nav>
+        <ul class="menu-items">
+          <li class="item">
+            <a
+              class="list is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="item item-3">
+            <a
+              class="list"
+              href="#"
+            >
+              samsung
```

Comment:
The word 'samsung' should be capitalized to 'Samsung' to maintain consistency and professionalism in the navigation menu.